astro/src/content/docs/get-started/core-concepts/collected-metrics.mdx

```diff
@@ -0,0 +1,535 @@
+---
+title: Collected Metrics
+description: An overview of how, when, and why FusionAuth collects deployment usage data.
+section: get started
+subcategory: download and install
```

Comment:
Why is it in this subcategory? 

Should it be in both download and install and run in the cloud (like the setup wizard)?

Reply:
Or maybe in the reference section? 

In the cloud, we collect these metrics but don't let people disable them. But we still should tell them what we collect.

Reply:
Add to both download and install, and run in the cloud. D&I should be the canonical. Look at Setup Wizard in first time setup. 

Change current Collected Metrics. We collect metrics. Network access. Community version can block. If you have a licensed version. If you want to learn more <link>

IF we do code version: it goes under docs_shared